test/helpers/SVGContainer.js

```diff
@@ -15,24 +15,26 @@
  * == BSD2 LICENSE ==
  */
 
-.common {
-  composes: svgMiddleAnchored svgVerticalCentered from '../../../styles/typography.css';
-  font-size: 12px;
-  pointer-events: none;
-}
+import React, { PropTypes } from 'react';
 
-.text {
-  composes: common;
-  composes: highContrastText from '../../../styles/typography.css';
-  font-weight: bold;
-}
+const SVGContainer = (props) => {
+  const { children, dimensions: { width, height } } = props;
 
-.median {
-  composes: common;
-  composes: whiteText from '../../../styles/typography.css';
-}
+  return (
+    <div>
+      <svg width={width} height={height}>
+        {children}
+      </svg>
+    </div>
+  );
+};
 
-.medianAlone {
-  composes: median;
-  font-weight: bold;
-}
\ No newline at end of file
+SVGContainer.propTypes = {
+  children: PropTypes.object.isRequired,
```

Comment:
As discussed earlier, this is a test helper that I don't feel needs to be quite as thoroughly documented...

@krystophv let me know if you disagree.